pymc/tests/distributions/test_timeseries.py

```diff
@@ -853,9 +856,9 @@ def sde_fn(x, k, d, s):
                 "y", dt=0.02, sde_fn=sde_fn, sde_pars=sde_pars, init_dist=init_dist, **kwargs
             )
 
-        y_eval = draw(y, draws=2)
+        y_eval = draw(y, draws=2, random_seed=RANDOM_SEED)
```

Comment:
Suggestion: you can pass the `numpy_rng` as `random_seed`

Reply:
done